lab/lab2/part1.js

```diff
@@ -18,62 +18,62 @@ console.log('Nathan\'s list', nathanGameList);
 What is the first game in Ross's list?
 ===================== */
 
-var query1;
+var query1=_.first(rossGameList);
 
 console.log('What is the first game in Ross\'s list?', query1);
 
 /* =====================
 What are all of the games except for the first game in ross's list?
 ===================== */
 
-var query2;
+var query2= _.rest(rossGameList);
 
 console.log('What are all of the games except for the first game in Ross\'s list?', query2);
 
 /* =====================
 What is the last game in Nathan's list?
 ===================== */
 
-var query3;
+var query3=_.last(nathanGameList);
 
 console.log('What is the last game in Nathan\'s list?', query3);
 
 /* =====================
 What are all of the games in Nathan's list except for the last?
 ===================== */
 
-var query4;
+var query4=_.initial(nathanGameList);
 
 console.log('What are all of the games in Nathan\'s list except for the last?', query4);
 
 /* =====================
 What would Nathan's game list look like if he sold "catan"?
 ===================== */
 
-var query5;
+var query5= _.without(nathanGameList, "catan");
 
 console.log('What would Nathan\'s game list look like if he sold "catan"?', query5);
 
 /* =====================
 If Nathan and Ross play a board game, what are their options? This should be a list of all games owned by ross or Nathan, with no duplicates.
 ===================== */
 
-var query6;
+var query6= _.union(nathanGameList,rossGameList);
 
 console.log('If Nathan and Ross play a board game, what are their options? This should be a list of all games owned by ross or Nathan, with no duplicates.', query6);
 
 /* =====================
 Which games are owned by both Ross and Nathan?
 ===================== */
 
-var query7;
+var query7 = _.intersection(nathanGameList,rossGameList);
 
 console.log('Which games are owned by both Ross and Nathan', query7);
 
 /* =====================
 Which games are exclusive to collections? In other words, only owned by either Ross or Nathan.
 ===================== */
 
-var query8;
+var query8= _.difference(nathanGameList,rossGameList);
```

Comment:
`difference` is insufficient here. 
This only shows nathans list minus my list, but it doesn't have the ones I have, that nathan doesn't have.